producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -70,143 +76,137 @@ public class SdtEndpointPortTypeTest extends AbstractSdtUnitTestBase {
+    private static final String RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN = "Runtime exception should have been thrown";
+    private static final String SDT_SYSTEM_COMPONENT_ERROR =
+            "A SDT system component error has occurred. Please contact the SDT support team for assistance";
+
     /**
      * Set up common for all tests.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        portType = new SdtEndpointPortType();
+        MockitoAnnotations.openMocks(this);
 
-        mockCreateBulkRequestHandler = EasyMock.createMock(IWsCreateBulkRequestHandler.class);
+        portType = new SdtEndpointPortType();
         portType.setWsCreateBulkRequestHandler(mockCreateBulkRequestHandler);
 
-        mockBulkRequestHandler = EasyMock.createMock(IWsReadBulkRequestHandler.class);
         portType.setWsReadBulkRequestHandler(mockBulkRequestHandler);
 
-        mockSubmitQueryHandler = EasyMock.createMock(IWsReadSubmitQueryHandler.class);
         portType.setWsReadSubmitQueryHandler(mockSubmitQueryHandler);
     }
 
     /**
      * Test submit bulk method completes successfully.
      */
     @Test
-    public void testSubmitBulkSuccess() {
+    void testSubmitBulkSuccess() {
         final BulkResponseType dummyResponse = createBulkResponse();
         final BulkRequestType dummyRequest = createBulkRequest();
 
-        EasyMock.expect(mockCreateBulkRequestHandler.submitBulk(dummyRequest)).andReturn(dummyResponse);
-        EasyMock.replay(mockCreateBulkRequestHandler);
+        when(mockCreateBulkRequestHandler.submitBulk(dummyRequest)).thenReturn(dummyResponse);
 
         final BulkResponseType response = portType.submitBulk(dummyRequest);
 
-        EasyMock.verify(mockCreateBulkRequestHandler);
-        assertNotNull("Response expected", response);
+        verify(mockCreateBulkRequestHandler).submitBulk(dummyRequest);
+        assertNotNull(response, RESPONSE_EXPECTED);
     }
 
     /**
      * Test submit bulk method handles exceptions successfully.
      */
     @Test
-    public void testSubmitBulkException() {
-        EasyMock.expect(mockCreateBulkRequestHandler.submitBulk(EasyMock.anyObject(BulkRequestType.class)))
-                .andThrow(new RuntimeException("test"));
-        EasyMock.replay(mockCreateBulkRequestHandler);
+    void testSubmitBulkException() {
+        when(mockCreateBulkRequestHandler.submitBulk(any())).thenThrow(new RuntimeException("test"));
 
         try {
             portType.submitBulk(createBulkRequest());
-            fail("Runtime exception should have been thrown");
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
         } catch (final RuntimeException re) {
-            assertEquals("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");

```

Comment:
Might as well use version of assertEquals() that doesn't have the message parameter.